chart/gatekeeper-operator/generate_helm_template.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+scriptdir="$(dirname "$0")"
+cd "$scriptdir"
+cp ./../../deploy/gatekeeper.yaml ${PWD}/helm-modifications/_temp.yaml
+kustomize build helm-modifications -o templates/gatekeeper.yaml
+sed -i -E "s/HELMSUBST_STATEFUL_SET_RESOURCES/\
+\n{{ toYaml .Values.resources | indent 12 }}\
+\n    {{- with .Values.nodeSelector }}\
+\n      nodeSelector:\
+\n{{ toYaml . | indent 8 }}\
+\n    {{- end }}\
+\n    {{- with .Values.affinity }}\
+\n      affinity:\
+\n{{ toYaml . | indent 8 }}\
+\n    {{- end }}\
+\n    {{- with .Values.tolerations }}\
+\n      tolerations:\
+\n{{ toYaml . | indent 8 }}\
+\n    {{- end }}/" templates/gatekeeper.yaml
+sed -i "s/HELMSUBST_VALUES_REPLICAS_PLACEHOLDER/{{ .Values.Replicas }}/g" templates/gatekeeper.yaml
+sed -i "s/HELMSUBST_VALUES_AUDITINTERVAL_PLACEHOLDER/{{ .Values.auditInterval }}/g" templates/gatekeeper.yaml
+sed -i "s/HELMSUBST_VALUES_CONSTRAINTVIOLATIONSLIMIT_PLACEHOLDER/{{ .Values.constraintViolationsLimit }}/g" templates/gatekeeper.yaml
```

Comment:
Instead of doing these seds, can we update helm-modifications.yaml to use the parameters from values.yaml? 
Example:  
```
replicas:  {{ .Value.Replicas}}
```

Reply:
I hate doing it this way but kustomize was giving me an error if I put ``{{ .Values.Replicas }}`` without quotes around it (which then becomes an invalid value because it's a string rather than an integer):
```
Error: builtin PatchStrategicMergeTransformer config: [112 97 116 104 115 58 10 45 32 104 101 108 109 45 109 111 100 105 102 105 99 97 116 105 111 110 115 46 121 97 109 108 10]: YAML file [helm-modifications.yaml] encounters a format error.
error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{".Values.replicas":interface {}(nil)}
```
If you happen to know a way around that please let me know. Otherwise this was a messy workaround.

In any event it turns out this issue doesn't affect the other arg parameters so I removed those sed lines.